docs/install/Knative-with-MicroK8s.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Install on MicroK8s"
+linkTitle: "MicroK8s"
+weight: 10
+type: "docs"
+---
+
+This guide walks you through the installation of Knative using [MicroK8s](https://microk8s.io).
+
+You can find [guides for other platforms here](./README.md).
+
+### Install MicroK8s
+
+```shell
+   sudo snap install --classic microk8s
```

Comment:
It should be clear that an Ubuntu system with Snap installed is a pre-requisite.

Reply:
Added details.